vlm/src/test/scala/geotrellis/contrib/vlm/avro/GeotrellisRasterSourceSpec.scala

```diff
@@ -31,9 +31,11 @@ import geotrellis.vector.Extent
 import org.scalatest.{FunSpec, GivenWhenThen}
 
 class GeotrellisRasterSourceSpec extends FunSpec with RasterMatchers with BetterRasterMatchers with GivenWhenThen with CatalogTestEnvironment {
-  val uriMultiband = s"file://${TestCatalog.multibandOutputPath}"
-  val uriSingleband = s"file://${TestCatalog.singlebandOutputPath}"
   val layerId = LayerId("landsat", 0)
+  val params = s"?layer=${layerId.name}&zoom=${layerId.zoom}"
+  val uriMultibandNoParams = s"file://${TestCatalog.multibandOutputPath}"
+  val uriMultiband = s"file://${TestCatalog.multibandOutputPath}$params"
```

Comment:
Why do we need such URIs here, since the dispatch by layerId happens inside the `GeoTrellisRasterSource` (`LayerId` is passed as an extra constructor parameter).

Reply:
The `uriMultibandNoParams` val is for the `CollectionLayerReader` since that can't take `URI`s with query parameters.

Reply:
@jbouffard the question is in this:
```scala
val uriMultiband = s"file://${TestCatalog.multibandOutputPath}$params"
val uriSingleband = s"file://${TestCatalog.singlebandOutputPath}?layer=${layerId.name}&zoom=${layerId.zoom}"

lazy val sourceMultiband = new GeotrellisRasterSource(uriMultiband, layerId)
lazy val sourceSingleband = new GeotrellisRasterSource(uriSingleband, layerId)
```

Do we need such formated uris for `GeoTrellisRasterSources`? It looks like it doesn't use it at all. I though such URI is required for SPI only.

Reply:
Ah, I see what you're saying. Yeah, it's that way from now due to `GeoTrellisRasterSource` needing a `DataPath`. Though, now that I'm thinking about it, I'm not sure if that's really ideal :confused:

Reply:
redundant in this case ): can you fix it?

Reply:
Yeah, I can do that :+1: